modules/hitl/src/views/full/typing/index.jsx

```diff
@@ -11,15 +11,17 @@ export default class Typing extends React.Component {
     }
   }
 
-  handleChange = (event) => {
+  handleChange = event => {
     this.setState({
       message: event.target.value
     })
   }
 
-  handleKeyPress = (event) => {
-    if (event.key === 'Enter' && this.state.message) {
-      this.props.sendMessage(this.state.message)
+  handleKeyPress = event => {
+    const message = this.state.message.trim()
+
+    if (event.key === 'Enter' && message) {
```

Comment:
Thanks for your contribution! Could you add the null check before doing the trim ? Also, in your condition, you should check if message.length > 0 instead if it is null